hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.small
+9. plate,bento,plate .pickle
+10. \*
+11. fancy \*, plate \*
+12. plate + .small, plate + apple
+13. pickle ~ pickle
+14. plate > apple
+15. orange:first-child
+16. apple,plate pickle:only-child
+17. apple, pickle:last-child
+18. :nth-child(3)
+19. bento:nth-last-child(3)
+20. apple:first-of-type
+21. :nth-of-type(even)
+22. :nth-of-type(2n+3)
+23. apple.small:only-of-type
+24. orange:last-of-type, apple:last-of-type
+25. bento:empty
+26. :not(#fancy, .small, plate:nth-last-child(2), plate)
+27. for
+28. plate[for]
+29. bento[for="Vitaly"]
+30. [for^="Sam"], [for^="Sarah"]
+31. [for$="Hayato"], [for$="Minato"]
+32. [for*="Bobby"], [for*="Robbie"]
```

Comment:
I think ^ means "start with", so since both Sam and Sarah starts with Sa, we can use `[for^="Sa"]`
Same for the last 2, `[for$="ato"] and [for*="obb"]`

Reply:
This way looks more polished. Thank you!